libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -203,13 +205,14 @@ void AP_InertialSensor_Backend::_notify_new_accel_raw_sample(uint8_t instance,
         _sem->give();
     }
 
+    // log raw data if flag is set
```

Comment:
This comment isn't great. The code is dependent on having the dataflash object or not - the fact that the function to get the dataflash object depends on a flag doesn't really matter here, that's exposing implementation details.